src/app/GitCommands/UserRepositoryHistory/RepositoryDescriptionProvider.cs

```diff
@@ -65,6 +77,34 @@ public string Get(string repositoryDir)
             return dirInfo.Name;
         }
 
+        public string GetDescriptiveUnique(string repositoryDir)
+        {
+            string unique = GetUnique(repositoryDir);
```

Comment:
path set as `C__` when :\ is replaced in the filename. Maybe explicit rename here to avoid two __